environment.yml

```diff
@@ -12,6 +12,8 @@ dependencies:
 - anaconda::setuptools=27.2.0
 - anaconda::statsmodels=0.8.0
 - conda-forge::altair=1.2.0
+- conda-forge::dask-searchcv
```

Comment:
I added version numbers to both `dask` and `dask-searchcv` and used anaconda for `dask`. Everything works fine.